api/src/api/agencies_v1/agency_schema.py

```diff
@@ -0,0 +1,77 @@
+from src.api.schemas.extension import Schema, fields
+from src.api.schemas.response_schema import AbstractResponseSchema, PaginationMixinSchema
+from src.pagination.pagination_schema import generate_pagination_schema
+
+
+class AgencyFilterV1Schema(Schema):
+    agency_id = fields.Integer()
+
+
+class AgencyListRequestSchema(Schema):
+    filters = fields.Nested(AgencyFilterV1Schema())
+    pagination = fields.Nested(
+        generate_pagination_schema(
+            "AgencyPaginationV1Schema",
+            ["created_at"],
+        ),
+        required=True,
+    )
+
+
+class AgencyContactInfoSchema(Schema):
+    """Schema for agency contact information"""
+
+    contact_name = fields.String()
+    address_line_1 = fields.String()
+    address_line_2 = fields.String(allow_none=True)
+    city = fields.String()
+    state = fields.String()
+    zip_code = fields.String()
+    phone_number = fields.String()
+    primary_email = fields.String()
+    secondary_email = fields.String(allow_none=True)
```

Comment:
Could you add some descriptions to these?

Reply:
Added!